glimpse/src/main/java/edu/asu/diging/glimpse/service/FileService.java

```diff
@@ -0,0 +1,36 @@
+package edu.asu.diging.glimpse.service;
+
+import org.springframework.stereotype.Service;
+import org.springframework.web.multipart.MultipartFile;
+ 
+import java.io.File;
+import java.io.IOException;
+ 
+@Service("fileService")
+public class FileService {
+ 
+    public static final String SAVE_LOCATION = "C:/SpringUpload/";
+ 
+    public boolean saveFile(MultipartFile multipartFile){
+        boolean result = false;
+        //set the saved location and create a directory location
+        String fileName  = multipartFile.getOriginalFilename();
+        String location = SAVE_LOCATION;

```

Comment:
if location == SAVE_LOCATION, why using an extra variable in the first place? why not use SAVE_LOCATiON directly?